packages/components/loading/src/loading-provider.tsx

```diff
@@ -359,7 +359,10 @@ const getOverlayStyle = (
   bottom: 0,
   left: 0,
   zIndex: 9999,
-  bg: type === "fill" ? ["#fbfbfb", "#141414"] : "rgba(0, 0, 0, 0.48)",
+  bg:
+    type === "fill"
+      ? ["fallback(white, #fbfbfb)", "fallback(black, #141414)"]
+      : "fallback(blackAlpha.600, rgba(0, 0, 0, 0.48))",
```

Comment:
@hirotomoyamada 
Is it safe to ignore these warnings?

Reply:
What's warning?